pop-api/README.md

```diff
@@ -1 +1,73 @@
-Reserved crate for pop-api.
\ No newline at end of file
+# Pop API
+
+## Examples
+ink! smart contract examples using the Pop API
+- [balance-transfer](./examples/balance-transfer/)
+- [NFTs](./examples/nfts/)
+- [place-spot-order](./examples/place-spot-order/)
+- [read-runtime-state](./examples/read-runtime-state/)
+
+## What is the Pop API?
+
+One of the core value propositions of Pop Network is to enable smart contracts to easily access the power of Polkadot. As such, the Pop API was built to enable smart contracts to easily utilize the functionality provided by the Pop Network parachain runtime.
+
+Substrate already exposes a Runtime API which is typically associated with the “outer node” calling into the runtime via RPC. Pop Network extends this concept in a direction that makes it usable by smart contracts, allowing untrusted contracts to make use of carefully exposed trusted functionality available within the runtime - this is the Pop API.
+
+The Pop API is designed to be:
+- **Stable**: contracts are generally immutable and therefore need a stable interface, especially as the runtime evolves over time.
+- **Future Proof**: technologies improve, so the Pop API implementation is adaptable to new approaches without affecting the interface. The Pop API implementation also selects the optimal approach to realize a use case from a range of possible technical implementations.
+- **Versioned**: not every future technology can be retro-fitted, so the API is versioned.
+- **Simple**: providing an abstraction layer over runtime-level features. Hand-rolled with love, Pop API is easy to use for smart contract development.
+
+The Pop API consists of three main parts:
+- the Pop API ink! library
+- the Pop API runtime code (chain extension)
+- shared primitive types between the Pop API ink! library and the Pop API runtime code
+
+Let's go over the flow.
+
+## The Pop API ink! Library
+Everything in [`pop-api`](./src/) **is the** Pop API ink! library.
+
+So when the ink! smart contract wants to use the Pop API library, it can simply have a line like:
+```rust
+use pop_api::nfts;
+```
+
+## The Glue
```

Comment:
Remove section. Not very important to understand for the smart contract developer and not theoretically true since it is much more than just the types in primitives

Reply:
I disagree. This is part is essential. It is the bridge btwn ink! pop-api and runtime pop-api.

Reply:
These types can also be found in the pop api so you would have to add that then.